docs/user_guide/llms/text_classification.md

```diff
@@ -0,0 +1,35 @@
+Pretrained LLMs are available as text encoders for general text features, and can be included in ECD models for binary or multi-class text classification tasks.
+
+The LLM encoder shares most of its features with the LLM model type, including base model selection, adapters, quantization, and initialization parameters like RoPE scaling. Unlike the LLM model type, the LLM encoder is part of an ECD architecture and does not generate text directly. Instead the input text is processed by the LLM and the final hidden state is passed forward to the combiner and decoder(s), allowing it to be used for predictive tasks directly.
```

Comment:
Along with RoPE scaling, can we also add `NEFTune embeddings`

Reply:
I see it's missing docs, so will try and add that

Reply:
Yeah! Happy to add it in when the docs are in place.

Reply:
Thanks! I can also add it in, let's get this merged first :) I was under the impression that the docs were already there for some reason.